website/versioned_docs/version-2.x/guides/compression.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 5
+---
+
+# Compression
+
+In this section, we will learn how to configure Producer Compression in KafkaFlow.
+
+KafkaFlow relies on the native message compression provided by the [Confluent Kafka client](https://github.com/confluentinc/confluent-kafka-dotnet). 
+
+The following compression types are supported:
+* Gzip
+* Snappy
+* Lz4
+* Zstd
+
+```csharp
+.WithCompression(CompressionType.Gzip)
+```
+
+:::info
+If you want to use a different compression type, visit the [Compressor Middleware guide](middlewares/compressor-middleware).
+:::
+
+Optionally, it's possible to specify the compression level, providing it as the second argument.
+You can find the possible values [here](https://docs.confluent.io/platform/current/clients/confluent-kafka-dotnet/_site/api/Confluent.Kafka.ProducerConfig.html#Confluent_Kafka_ProducerConfig_CompressionLevel).
+
+```csharp
+.WithCompression(CompressionType.Gzip, 5)
+```
+
+:::info
```

Comment:
I've noticed that this kind of tags are broken on this documentation. Also I've checked if on master an it does too [here](https://github.com/Farfetch/kafkaflow/blob/master/website/docs/guides/compression.md?plain=1#L21). 

Is it relevant to fix this at this moment?

Reply:
Those are macros for docusauros that aren't rendered in github markdown.